amundsen_application/static/js/components/TableDetail/TableIssues/index.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import { GlobalState } from 'ducks/rootReducer';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import { Issue } from 'interfaces'; 
+import { getIssues } from 'ducks/issue/reducer'; 
+import { logClick } from 'ducks/utilMethods';
+import { GetIssuesRequest } from 'ducks/issue/types';
+import './styles.scss';
+import { 
+  ASSOCIATION_TEXT, 
+  ADDITIONAL_ISSUES_TEXT, 
+  ADDITIONAL_ISSUES_END, 
+  ADDITIONAL_ISSUES_START
+} from './constants';
+
+
+export interface StateFromProps {
+  issues: Issue[]; 
+  remainingIssues: number; 
+  remainingIssuesUrl: string; 
+}
+
+export interface DispatchFromProps {
+  getIssues: (key: string) => GetIssuesRequest; 
+}
+
+export interface ComponentProps {
+  tableKey: string;
+}
+
+export type TableIssueProps = StateFromProps & DispatchFromProps & ComponentProps; 
+
+export class TableIssues extends React.Component<TableIssueProps> {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    this.props.getIssues(this.props.tableKey);
+  }
+
+  renderIssue = (issue: Issue, index: number) => {
+    return (
+      <div className="issue-banner" key={`issue-${index}`}>
+        <a id={`table-issue-link-${index}`} className="table-issue-link" target="_blank" href={issue.url} onClick={logClick}>
+          <img className="icon icon-red-triangle-warning "/>
```

Comment:
Do all issues automatically get a `red-triangle-warning` or is there a way to map some sort of severity to different colors? Also do we display issue `status`?